webapp/src/modules/account/sagas.spec.ts

```diff
@@ -129,3 +146,176 @@ describe('when handling the request to fetch account metrics', () => {
     })
   })
 })
+
+describe('when handling the request to fetch creators accounts', () => {
+  let search: string
+  let accounts: Account[]
+  let addresses: string[]
+
+  describe('when a call to the nftAPI fails', () => {
```

Comment:
What if we fuse the two describes into one?
```suggestion
  describe('when the request with a search term fails while fetching the NFT API', () => {
```